Bottom: ae67715850c1e4254aeeb6d7eba594746b80123d
Top:    1622a70dcb0ee968dd8c0302d5f9516c76b7f27c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-19 22:57:07 +0000

Refresh of add-hidgc-parameter-to-various

---

diff --git a/src/print.c b/src/print.c
index d494007..27e1c98 100644
--- a/src/print.c
+++ b/src/print.c
@@ -111,7 +111,7 @@ text_at (hidGC gc, int x, int y, int align, char *fmt, ...)
   t.X -= w * (align & 3) / 2;
   if (t.X < 0)
     t.X = 0;
-  DrawTextLowLevel (&t, 0);
+  DrawTextLowLevel (gc, &t, 0);
   if (align & 8)
     fab_line (gc, t.X,
               t.Y + SCALE_TEXT (font->MaxHeight, t.Scale) + MIL_TO_COORD(10),
@@ -346,7 +346,7 @@ PrintFab (hidGC gc)
       END_LOOP;
       TEXT_LOOP (layer);
       {
-	DrawTextLowLevel (text, 0);
+	DrawTextLowLevel (gc, text, 0);
       }
       END_LOOP;
       gui->graphics->set_line_width (gc, FAB_LINE_W);
